### PR TITLE
implement card ui

### PR DESCRIPTION
**Pull request by SVyusti**
## Related Issue

 Implement Card Component for Internships 

Closes: #416

- Added a card component for featuring internships using material UI and fixed text overflow.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.
Screenshot from 2023-06-23 20-41-20
Screenshot from 2023-06-23 20-41-35

### CODE REVIEW

**Review by anmode — CHANGES_REQUESTED** [review]
we have tailwind! 
please avoid using material UI

**Comment by SVyusti** [comment]
Can i use nextui-org for card

**Comment by anmode** [comment]
@SVyusti The thing is we want to have optimized card... we have card component you can improve the card by using tailwind css.

No worries take your time I will email back that you have contributed to amupedia and created some issues in grabtern and also raised a PR.

**Comment by SVyusti** [comment]
Thank you, 
Pls review changes.

File: components/Internship.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import Image from "next/image";
+import {Image,Card,Text,Link} from "@nextui-org/react"
+
```
Comment (anmode) [inline components/Internship.js:3]: revert this if not needed!

File: package.json [hunk 2]
```diff
@@ -11,6 +11,11 @@
   },
   "dependencies": {
     "@emailjs/browser": "^3.10.0",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/joy": "^5.0.0-alpha.85",
+    "@mui/material": "^5.13.6",
+    "@nextui-org/react": "^1.0.0-beta.13",
```
Comment (anmode) [inline package.json:18]: revert this!

File: public/assets/css/style.css [hunk 3]
```diff
@@ -3364,12 +3364,51 @@ hr {
...
+}
+.courses-area .properties__link:hover{
+  color: #4255a4;
+  text-Decoration: underline;
+  font-Size:2rem;
+}
+.courses-area .properties__info{
+  max-height:13rem; 
+  overflow-y: hidden;
+  text-overflow: ellipsis;
```
Comment (anmode) [inline public/assets/css/style.css:3391]: move all css of internship card in a css file named intership-section
and please update css names from courses -> internship
we can have css file in `styles` folder

**Review by anmode — CHANGES_REQUESTED** [review]
Thanks left some reviews!
But what i noticed.. carousel is not behaving properly here how you can test:
 see production one and your preview
2. Card image takes more image and description as aspect ratio is not look great

**Pull request closed without merging**